eq-author-api/README.md

```diff
@@ -156,3 +141,21 @@ Add the following to your `launch.json` configuration:
 ```
 
 Then start your tests [as described above](#tests). You can now start a debugging session, and pick the jest process to attach to.
+
+## DynamoDB
+
+### Running locally
+
+To run DynamoDB locally, ensure that you are in the eq-author-api folder and then run:
+
+    yarn dynamodb-admin
```

Comment:
To clarify, this command does not run DynamoDB locally. DynamoDB is run as part of the `docker-compose up` command. See config in `docker-compose.yml`. The above command is a separate tool that allows developers to inspect the data held within Dynamo during local development.

Reply:
This has been resolved now.